docs/source/cli/xcube_serve.rst

```diff
@@ -143,42 +140,52 @@ and the it is referred to by the non-hidden, actual dataset using the ``TimeSeri
 
       - Identifier: my_dataset
         Title: "My Dataset"
-        FileSystem: local
+        FileSystem: file
         Path: my_dataset.zarr
         TimeSeriesDataset: my_dataset_opt_for_ts
 
       - Identifier: my_dataset_opt_for_ts
         Title: "My Dataset optimized for Time-Series"
-        FileSystem: local
+        FileSystem: file
         Path: my_ts_opt_dataset.zarr
-        Format: zarr
         Hidden: True
 
       - ...
 
 
+.. _config:
 Server Demo Configuration File
 ==============================
 The server configuration file consists of various parts, some of them are necessary others are optional.
-Here the `demo configuration file`_ used in the :ref:`example` is explained in detail.
+Here the `demo configuration file`_ used in the `example`_ is explained in detail.
```

Comment:
The example link does not work as you have two examples in this file. Can you rename them appropriately?